test/unit/actions/action_with_sub_plans_test.rb

```diff
@@ -35,7 +35,7 @@ def run; end
         user = FactoryBot.create(:user)
         triggered = ForemanTasks.trigger(ParentAction, user)
         raise triggered.error if triggered.respond_to?(:error)
-        triggered.finished.wait(2)
+        triggered.finished.wait(30)
```

Comment:
Does this need to stay increased when the root cause was resolved?

Reply:
I would leave it there